.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -0,0 +1,53 @@
+---
+name: 🐛Bug Report🐛
+about: Report an Issue with the JDBC Driver
+title: "[BUG]"
+labels: 'bug, untriaged, Beta'
```

Comment:
Nit: Beta capitalization intended?

Reply:
We don't even have these labels actually. I am going to remove them entirely.